CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Liana daemon and GUI release notes
 
+## 7.0
+
+This release introduces support for Wizardsardine servers as an optional backend for the Liana GUI
+and Electrum server as an optional node interface for the Liana daemon.
+
+### Breaking changes
+
+Liana Daemon has a migration for its database to add a new table `transactions` to keep in cache
+transactions to reduce requests to the node. Once Liana GUI or `lianad` run for the first time
+this migration is applied and cannot be reverted.
+
+### Features
+
+#### Liana daemon / library
+
+- We added a anti fee snipping protection to the create spend transaction command.
+- We added optional support for an Electrum server as interface to the bitcoin blockchain instead
+  of Bitcoin Core.
+
+
+#### Liana GUI
+
+- The user does not have anymore to select a network at start up, the launcher looks for a installed
+  network in the following order: `mainnet`, `testnet`, `signet`, `regtest`.
+- The launcher reads the `settings.json` file and provide information about the installed wallet or
+  provide the user a choice to create or add a wallet.
+- When creating or adding a wallet for the selected network, user has the choice to use his local
+  node or use the Wizardsardine servers has a backend for the Liana GUI.
+- User can share his hardware wallets xpubs by clicking on "Share xpubs" from the launcher.
+- Wallet policy description was added to the installer and in the wallet settings.
+- If using the Wizardsardine servers as remote backend, a user can provide access to a wallet to
+  an other user by sending him an invitation with the invitee email in the wallet settings.
+- Coldcard and Specter DIY can now be used for any taproot setup.
+- The software version is now in the title bar.
+- A `--help` flag was added to display a help menu about the available flags.
+- Bitcoin Core 27.1 is now downloaded if user chooses Liana managed node during install.
+
+### Fixes
+
+#### Liana daemon / library
+
+- We check that the change multipath is not hardened while importing descriptor in LianaDescriptor
+  wrapper.
+- We add the minimum feerate requested in the error message of the create spend command.
```

Comment:
I'm not sure to understand, did we add the requested feerate, or the minimum feerate?

Reply:
the minimum https://github.com/wizardsardine/liana/pull/1186